BookLibrary-cdi/src/main/java/org/booklibrary/app/rest/dto/BookDto.java

```diff
@@ -0,0 +1,46 @@
+package org.booklibrary.app.rest.dto;
+
+import org.booklibrary.app.persistence.entity.Book;
+import org.jboss.resteasy.annotations.providers.jaxb.json.Mapped;
+import org.jboss.resteasy.annotations.providers.jaxb.json.XmlNsMap;
+import org.jboss.resteasy.links.RESTServiceDiscovery;
+
+import javax.xml.bind.annotation.*;
+import java.io.Serializable;
+
+@Mapped(namespaceMap = @XmlNsMap(jsonName = "atom", namespace = "http://www.w3.org/2005/Atom"))
+@XmlRootElement(name = "book")
+@XmlAccessorType(XmlAccessType.NONE)
```

Comment:
**Consider using XmlAccessorType(PUBLIC_MEMBER)**:
Every public getter/setter pair and every public field will be automatically bound to XML, unless annotated by XmlTransient. Fields or getter/setter pairs that are private, protected, or defaulted to package-only access are bound to XML only when they are explicitly annotated by the appropriate JAXB annotations.